deploy/chart/templates/0000_50_olm_17-upstream-operators.catalogsource.yaml

```diff
@@ -0,0 +1,14 @@
+#! validate-crd: ./deploy/chart/templates/05-catalogsource.crd.yaml
+#! parse-kind: CatalogSource
+{{ if (.Values.installType) and eq .Values.installType "upstream" }}
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: operatorhubio-catalog
+  namespace: {{ .Values.catalog_namespace }}
+spec:
+  sourceType: grpc
+  image: quay.io/operator-framework/upstream-community-operators:latest
```

Comment:
This is good. I just wonder if we want to inject the version/digest as part of the template instead of hard-coding `latest`.